docs/tool-configuration/configuration-cloudformation.md

```diff
@@ -20,113 +20,41 @@ cloudformation:
 
 ## CLI Configuration
 
--------------------
-### validate-cfn
-* **BitOps Property:** `validate-cfn`
-* **Environment Variable:** `CFN_TEMPLATE_VALIDATION`
-* **default:** `true`
-
-Calls `aws cloudformation validate-template` 
-
--------------------
-### cfn-stack-action
-* **BitOps Property:** `cfn-stack-action`
-* **Environment Variable:** `CFN_STACK_ACTION`
-* **default:** `deploy`
-
-Controls what cloudformation action to apply on the stack
-
--------------------
+| Property         | Environmental Variable | Description                                               | Default  | Required |
+| ---------------- | ---------------------- | --------------------------------------------------------- | -------- | -------- |
+| validate-cfn     | FN_TEMPLATE_VALIDATION | Calls `aws cloudformation validate-template`              | `true`   |          |
+| cfn-stack-action | CFN_STACK_ACTION       | Controls what CloudFormation action to apply on the stack | `deploy` |          |
```

Comment:
Something is broken here with formatting
![image](https://user-images.githubusercontent.com/1533818/199078131-9402dcee-5ff4-4899-96cd-c26cafd3911a.png)

Reply:
Fixed